src/Sylius/Behat/Context/Api/Admin/ManagingProductsContext.php

```diff
@@ -94,6 +132,72 @@ public function iAddIt(): void
         $this->client->create();
     }
 
+    /**
+     * @When I add the :productOption option to it
+     */
+    public function iAddTheOptionToIt(ProductOption $productOption): void
+    {
+        $this
+            ->client
+            ->updateRequestData(['options' => ['/new-api/product_options/' . $productOption->getCode()]]);
```

Comment:
```suggestion
            ->updateRequestData(['options' => ['/new-api/product_options/' . $productOption->getCode()]])
        ;
```

Reply:
Will it add this option to the previous ones or replace them?

Reply:
Replace ;( Have you any idea how to fix it?

Reply:
Maybe one of the following will work:
1. try to send a PATCH request
2. fetch the previous data, collect it, add one option this content and send it back in put
3. treat options as a separate collection and allow for post and delete methods there?

Reply:
PATCH request also replace all elements in collection

Reply:
I open new PR for it after merge